src/parser/jobs/pct/modules/Swiftcast.tsx

```diff
@@ -0,0 +1,8 @@
+import {Trans} from '@lingui/macro'
+import {DataLink} from 'components/ui/DbLink'
+import {Swiftcast as CoreSwiftcast} from 'parser/core/modules/Swiftcast'
+import React from 'react'
+
+export class Swiftcast extends CoreSwiftcast {
+	override suggestionContent = <Trans id="pct.swiftcast.missed.suggestion.content">Cast a spell with <DataLink action="SWIFTCAST" /> before it expires. This allows you to instantly cast spells with a cast times, such as a motif for a long movement or weaving window.</Trans>
```

Comment:
This seems really clunky, beyond just "a cast times".  Maybe "This allows you to cast spells that have cast times instantly, such as using Swiftcast on a motif for a long movement or weaving window"?

Reply:
I didn't look too closely at the copy, since I'd cloned it from one of the healer ones, but yeah, that needed some work. I updated it and added the second example of using it to sneak a Rainbow Drip in before a downtime/end of fight.